tool/dart_model_generator/lib/generate_dart_model.dart

```diff
@@ -466,6 +481,7 @@ class ClassTypeDefinition implements Definition {
       result.writeln('  $name() : ');
     } else {
       result.writeln('  $name({');
+      // TODO: Why are we excluding Map properties?
```

Comment:
It's to stop people passing them in as SDK maps that we then have to copy :) I'll send you a follow-up explaining.